db/migrate/20190220183951_add_pit_deadline_extended_property_to_cases.rb

```diff
@@ -0,0 +1,17 @@
+class AddPitDeadlineExtendedPropertyToCases < ActiveRecord::Migration[5.0]
+  def up
+    Case::Base.unscoped.all.each do |kase|
+      pit_transitions = kase.transitions
+        .where(event: ['extend_for_pit', 'remove_pit_extension'])
+        .order(created_at: :desc)
+
+      if pit_transitions.any? && pit_transitions.last.event == 'extend_for_pit'
+        kase.has_pit_extension = true
+        kase.save!
+      end
+    end
+  end
+
+  def down
```

Comment:
We should raise something here if there is no down method so we know that this is an irreversible migration

Reply:
Decided against raising an exception because this migration creates the denormalisation as a JSON property, so if the database were to be rolled back the case_transitions would still exist, so there's nothing to roll back in theory.

Reply:
Should probably be a data migration then.